shared/actions/chat/inbox.js

```diff
@@ -105,6 +105,15 @@ function* onInboxStale(): SagaGenerator<any, any> {
     )
 
     const incoming = yield loadInboxChanMap.race()
+
+    if (incoming.finished) {
```

Comment:
we should handle this case anyways, we're racing and finish is a possible outcome so lets deal with it